evalml/pipelines/components/transformers/preprocessing/featuretools.py

```diff
@@ -36,16 +38,21 @@ def _make_entity_set(self, X):
         ft_es = EntitySet()
         # TODO: This delete was introduced for compatibility with Featuretools 1.0.0.  This should
         # be removed after Featuretools handles unnamed dataframes being passed to this function.
+        # But even then, we should still use any dataframe name that might be available from input features.
```

Comment:
The way I see it, if a user has passed `Feature` objects in, then we _do_ have a dataframe_name that we can use for `X` instead of `X.ww.name` (even if they're the same!), so this TODO explaining why we delete the woodwork schema does still apply.

I wanted to make sure that was clear, since I think it's a little less obvious now that we aren't fully hardcoding `"X"`.

Reply:
love it

Reply:
@jeremyliweishih added it 4c374086d - let me know if that makes sense to you!